### PR TITLE
Added Web Dashboard Report

### PR DESCRIPTION
**Pull request by kkrampa**
@czue 
web_dashboard

### CODE REVIEW

**Comment by czue** [comment]
wow, this looks great!

**Comment by kkrampa** [comment]
:white_check_mark:

File: custom/enikshay/reports/web_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        locations_id = [
+            Choice(value=location_id, display='') for location_id in self.report_config.locations_id
+            if location_id
+        ]
+
+        if locations_id:
+            filter_values['village'] = locations_id
+
+        sputum_conversion_report.set_filter_values(filter_values)
+        sputum_conversion_data = sputum_conversion_report.get_data()[0]
```
Comment (czue) [inline custom/enikshay/reports/web_dashboard.py]: nit: everything from `sputum_conversion_report = ReportFactory.from_spec(...` to here might be a good candidate to get pulled out into a helper function

File: custom/enikshay/reports/web_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        return ChartsSqlData(config=self.report_config).get_data()
+
+    @property
+    @memoized
+    def treatment_outcome_sql_data(self):
+        return TreatmentOutcomeSqlData(config=self.report_config).get_data()
+
+    @property
+    def charts(self):
+        case_finding_sql_data = self.case_finding_sql_data[0]
```
Comment (czue) [inline custom/enikshay/reports/web_dashboard.py]: for all of these does `get_data` always return only 1 row?

File: custom/enikshay/reports/web_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+        relapse_cases = case_finding_sql_data.get('recurrent_positive_tb', default_value)['sort_key']
+        failure_cases = case_finding_sql_data.get('failure_positive_tb', default_value)['sort_key']
+        lfu_cases = case_finding_sql_data.get('lfu_positive_tb', default_value)['sort_key']
+        others_cases = case_finding_sql_data.get('others_positive_tb', default_value)['sort_key']
+
+        chart2.add_dataset(
+            _('New'),
+            [
+                {'x': 'Smear +ve', 'y': positive_smear},
```
Comment (czue) [inline custom/enikshay/reports/web_dashboard.py]: nit: some of these missing translations?

**Review by czue — APPROVED** [review]
couple minor comments. looks good

**Pull request merged**